packages/wonder-blocks-dropdown/util/constants.js

```diff
@@ -14,3 +14,8 @@ export const selectDropdownStyle = {
     marginTop: Spacing.xSmall,
     marginBottom: Spacing.xSmall,
 };
+
+export const filterableDropdownStyle = {
+    minHeight: 100,
+    maxHeight: 384,
```

Comment:
Where do these constants come from? How were they determined? Can you add a comment, perhaps?

Reply:
Hi Jeff! The constants are from the design spec, "Dimensions" in the Figma. I wonder if adding "These numbers are from design" or "See dimensions from Figma <link>" would be helpful comment.

Reply:
I think commenting on the origins of these numbers would be helpful; just something that helps others know where they come from so that if we are considering changing them in future, we have context.